pkgs/tools/graphics/vulkan-tools/default.nix

```diff
@@ -64,14 +66,13 @@ stdenv.mkDerivation rec {
 
   dontPatchELF = true;
 
-  env.PKG_CONFIG_WAYLAND_SCANNER_WAYLAND_SCANNER="${buildPackages.wayland-scanner}/bin/wayland-scanner";
+  env.PKG_CONFIG_WAYLAND_SCANNER_WAYLAND_SCANNER = "${lib.getBin buildPackages.wayland-scanner}/bin/wayland-scanner";
```

Comment:
This is a simple `getExe` now, but I should stop bike-shedding

Reply:
bike sheds addressed.